coral-component-multifield/src/scripts/Multifield.js

```diff
@@ -246,6 +246,13 @@ const Multifield = Decorator(class extends BaseComponent(HTMLElement) {
         this.trigger('change');
 
         this._trackEvent('click', 'add item button', event);
+
+        // Focus the newly created input
+        const items = this.items.getAll();
+        const setsize = items.length;
+        const itemToFocus = items[setsize - 1];
+        const inputItem = itemToFocus.querySelector("coral-multifield-item-content > input");
```

Comment:
The multifield component support any template, it's not necessary that its will always contain an input field.

Reply:
You can use `commons.TABBABLE_ELEMENT_SELECTOR` to cover different types of elements.